inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx

```diff
@@ -1434,11 +1434,10 @@ cdef class IENetwork:
     #   ```
 
     def __cinit__(self, model: [str, bytes] = "", weights: [str, bytes] = "", init_from_buffer: bool = False):
-        # TODO: ucomment when ngraph python api will work
         # Try to create Inference Engine network from capsule
-        # if model.__class__.__name__ == 'PyCapsule' and weights == '' and init_from_buffer is False:
-        #     self.impl = C.IENetwork(model)
-        #     return
+        if model.__class__.__name__ == 'PyCapsule' and weights == '' and init_from_buffer is False:
+            self.impl = C.IENetwork(model)
+            return
```

Comment:
get_function() is still under comments in this file. I suppose it should be enabled as well.

Reply:
I will do this in the next PR, related to ONNX tests.